### ISSUE TITLE
ModuleNotFoundError on import in latest release 0.3.12

### ISSUE THREAD

**Issue by pgandhi-delphix** [pycontribs/jenkinsapi#833]
##### ISSUE TYPE

 - Bug Report

##### SUMMARY
The latest release 0.3.12 results in a `ModuleNotFoundError`:
```
 $ pip show jenkinsapi
Name: jenkinsapi
Version: 0.3.12
Summary: A Python API for accessing resources on a Jenkins continuous-integration server.
Home-page: None
Author: None
Author-email: Salim Fadhley <salimfadhley@gmail.com>, Aleksey Maksimov <ctpeko3a@gmail.com>
License: None
Location: .../jenkinsapi/lib/python3.8/site-packages
Requires: six, pytz, requests
Required-by:

 $ python
Python 3.8.10 (v3.8.10:3d8993a744, May  3 2021, 09:09:08)
[Clang 12.0.5 (clang-1205.0.22.9)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
f>>> from jenkinsapi import jenkins
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../jenkinsapi/lib/python3.8/site-packages/jenkinsapi/__init__.py", line 50, in <module>
    from jenkinsapi import (
  File ".../jenkinsapi/lib/python3.8/site-packages/jenkinsapi/api.py", line 15, in <module>
    from jenkinsapi.jenkins import Jenkins
  File ".../jenkinsapi/lib/python3.8/site-packages/jenkinsapi/jenkins.py", line 18, in <module>
    from jenkinsapi.jobs import Jobs
  File ".../jenkinsapi/lib/python3.8/site-packages/jenkinsapi/jobs.py", line 7, in <module>
    from jenkinsapi.job import Job
  File ".../jenkinsapi/lib/python3.8/site-packages/jenkinsapi/job.py", line 23, in <module>
    from jenkinsapi_utils.compat import to_string
ModuleNotFoundError: No module named 'jenkinsapi_utils'
```

**Comment by cedrak** [comment]
I confirm that this happens:
```
$ pip show jenkinsapi
Name: jenkinsapi
Version: 0.3.12
Summary: A Python API for accessing resources on a Jenkins continuous-integration server.
Home-page: 
Author: 
Author-email: Salim Fadhley <salimfadhley@gmail.com>, Aleksey Maksimov <ctpeko3a@gmail.com>
License: 
Location: ~/tmp/lib/python3.10/site-packages
Requires: pytz, requests, six
Required-by: 

$ python3
Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import jenkinsapi
[...]
ModuleNotFoundError: No module named 'jenkinsapi_utils'
```

**Comment by lechat** [comment]
I will fix that soon.

**Comment by lechat** [comment]
This is fixed on 0.3.13
